curriculum/challenges/english/25-front-end-development/lecture-accessible-media-elements/672a55eb7791559421ff0cd3.md

```diff
@@ -32,29 +33,29 @@ To add captions or subtitles to your video or audio content, you can use the `tr
 <!-- Video -->
 <video controls src="video.mp4">
   <track
-      src="captions.vtt" 
-      kind="captions"
-      srclang="en"
-      label="English" 
+    src="captions.vtt" 
+    kind="captions"
+    srclang="en"
+    label="English" 
   />
 </video>
 
 <!-- Audio -->
 <audio controls src="audio.mp3">
-   <track
-       src="captions.vtt"
-       kind="captions"
-       srclang="en"
-       label="English"
-   />
+  <track
+    src="captions.vtt"
+    kind="captions"
+    srclang="en"
+    label="English"
+  />
 </audio>
 ```
 
 The `kind` attribute is used to tell the track element how it should be used. Valid values for the `kind` attribute include `captions`, `subtitles`, `chapters`, and `metadata`.
 
 The `srclang` attribute represents the language for the `track` content. The `label` attribute is a descriptive title for the text track that browsers use to identify and display it in the list of available text tracks.
 
-Another important thing to consider is providing a transcript for your audio and video content. A transcript is a text version of all the spoken words in your audio or video. Unlike captions, transcripts don’t need to be synchronized with the media. Transcripts are useful for deaf people and those hard of hearing. They're also beneficial for people who prefer reading instead of watching or listening. Transcripts also make your content searchable, allowing users to quickly find specific parts of your audio or video. If you have a video or audio on a website, you can simply add the captions below the audio or video:
+Another important thing to consider is providing a transcript for your audio and video content. A transcript is a text version of all the spoken words in your audio or video. Unlike captions, transcripts don’t need to be synchronized with the media. Transcripts are useful for deaf people and those hard of hearing. They're also beneficial for people who prefer reading instead of watching or listening. Transcripts also make your content searchable, allowing users to quickly find specific parts of your audio or video. If you have a video or audio on a website, you can simply add the transcript below the audio or video:
```

Comment:
Here is the issue for that 

https://github.com/freeCodeCamp/freeCodeCamp/issues/59814